docs/example-narrative-in-use-forecasts.md

```diff
@@ -16,65 +20,24 @@ curl -H "Accept: application/vnd.trolie.forecast-limit-set-slim.v1+json" \
 
 This will return the current version of the in-use ratings for the next 240 hours into output.json.  See the following for an example:
 
+### Reuse JSON method 1:
+
+ {% capture example %}
+ {{- site.data.paths["limits_forecast-snapshot"].get.responses["200"].content["application/vnd.trolie.forecast-limit-set-slim.v1+json"].example["$ref"] 
+    | split: "/" | last
+ -}}
+ {% endcapture %}
+
+{% assign example_json = '_data/examples/' | append: example %}
+
+```json
+{% include_relative {{example_json}} %}
+```
+
+### Method 2:
+
 ```json
-[
-    { 
-      "transmission-facility-id": "lineA",
-      "periods": [
-        {
-          "period-start": "2025-11-02T01:00:00-05:00",
-          "values": [
-            {
-             "type": "normal",
-             "value": {
-               "mva": 160
-             }
-            },
-            {
-             "type": "emergency",
-             "value": {
-                "mva": 165
-             }
-            }
-          ]
-        },
-        {
-            "period-start": "2025-11-02T01:00:00-06:00",
-            "values": [
-              {
-               "type": "normal",
-               "value": {
-                 "mva": 162
-               }
-              },
-              {
-               "type": "emergency",
-               "value": {
-                  "mva": 167
-               }
-              }
-            ]
-        }, 
-        {
-            "period-start": "2025-11-02T02:00:00-06:00",
-            "values": [
-              {
-               "type": "normal",
-               "value": {
-                 "mva": 162
-               }
-              },
-              {
-               "type": "emergency",
-               "value": {
-                  "mva": 167
-               }
-              }
-            ]
-        }              
-      ]
-    }
-]
+{% include_relative _data/examples/forecast-limits-slim.json %}
```

Comment:
This includes the same example, but unlike Method 1, requires the page to be in the top-level directory, because `include_relative` [does not support includes from the parent directory](https://jekyllrb.com/docs/includes/#including-files-relative-to-another-file).

That's why I named the file previous directory name as a prefix: `example-narrative...`. That would be one way to organize without using subfolders. Again, I don't love that, but it's an option.